docs/cudf/source/conf.py

```diff
@@ -44,6 +44,7 @@
     "IPython.sphinxext.ipython_directive",
     "nbsphinx",
     "recommonmark",
+    "sphinxemoji.sphinxemoji",
```

Comment:
Rather than adding this dependency, could we just use the unicode characters ✅and ❌ directly in the source? That's what I did in the groupby docs: https://raw.githubusercontent.com/rapidsai/cudf/branch-0.18/docs/cudf/source/groupby.md

Reply:
Thanks for this pointer @shwina ! I was able to utilize uni-code values instead of sphinxemoji in this case.